athlinks-schema.ttl

```diff
@@ -300,9 +300,8 @@ athlinks:participatedIn a owl:ObjectProperty ;
   rdfs:label "participated in"@en .
 
 athlinks:numberOfRaces a owl:DataTypeProperty ;
-  rdfs:domain schema:Person ;
-  rdfs:range xsd:unsignedInt .
   rdfs:comment "Number of races completed by an athlete." ;
   rdfs:label "Number of races" ;
-
+  rdfs:domain athlinks:Athlete ;
```

Comment:
Good catch on the period. But I think this should use schema:Person and not athlinks:Athlete, since we don't have any athlinks:Athlete objects in our db right now.